pages/documentation/developer/repositories/powsybl-open-loadflow.md

```diff
@@ -7,12 +7,10 @@ The PowSyBl Open Load Flow [repository](https://github.com/powsybl/powsybl-open-
 ## Features
 
 - a load flow solver
-- a security analysis (beta version)
```

Comment:
Why do you remove that?

Reply:
In fact I added that line by mistake in this PR, therefore I removed it later (thanks to previous review).